contracts/upgradeable_contracts/U_Validatable.sol

```diff
@@ -19,4 +20,24 @@ contract Validatable is EternalStorage {
         _;
     }
 
+    function setGasPrice(uint256 _gasPrice) public onlyOwner {
```

Comment:
Why do you put these calls to `Validatable` contract? It is not look logical for me. I would introduce a contract something like `BasicBridge` and move there this logic, management of tx and daily limits. Since it is common for both bridges.

Reply:
ok, lets rename validatable into BasicBridge?

Reply:
I would say that `Validatable` like `Ownable` for tokens.
At the same time tokens have some common part which is called `BasicToken`. So, I would use the same logic for bridge: differentiate logic related to validators and logic related to another bridge configuration.
But it is really `my liking` so up to you.